torch/_inductor/cpu_vec_isa.py

```diff
@@ -192,6 +192,8 @@ class VecAVX512(VecISA):
         if not _IS_WINDOWS
         else "/arch:AVX512"
     )  # TODO: use cflags
+    if not _IS_WINDOWS and torch._C._cpu._is_avx512_fp16_supported():
+        _arch_flags = _arch_flags + " -mavx512fp16"
```

Comment:
I feels it will break all the AVX512 usage for the GCC version doesn't support `-mavx512fp16`

Reply:
Oh, good point!